### ISSUE TITLE
"Some information has changed. Please review." is not announced to screenreaders.

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95542]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-1
**Page**: review-and-submit
**Environment**: any screenreader/browser combo
**Estimated Level of Effort to fix**: Medium
**Design system component involved**: Yes
**Relevant WCAG success criteria**: [4.1.3 Status Messages Level AA](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview#status-messages)

### To Reproduce:
- Use a screenreader and edit a question in order to trigger the "Some information has changed" state on the accordion button.

#### Expected Behavior
- Ideally the message is announced to screenreaders.

#### Actual Behavior
- There is no announcement. The user has to backtrack to find the message.

## Tasks
- [ ] Investigate whether we can throw an aria-live on the accordion headings and announce the change, or check out other options for sharing the update.
- [ ] If a fix is found implement. Otherwise document the issue.
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] Screenreader users hear "Some information has changed. Please review."
- [ ] Accessibility Audit issue tracking doc is updated with ticket status
